clearwater-queue-manager.root/usr/share/clearwater/clearwater-queue-manager/scripts/check_queue_state.py

```diff
@@ -47,5 +47,6 @@ def describe_queue_state():
         print "  Nodes that have completed:"
         for node in values["COMPLETED"]:
             print "    Node ID: {}".format(node["ID"])
-
+    
```

Comment:
Putting the formatting change here means that you've always got an extra line - even for cases where there's only one queue.